internal/features/stacks/events.go

```diff
@@ -200,9 +201,23 @@ func (f *StacksFeature) decodeStack(ctx context.Context, dir document.DirHandle,
 	}
 	ids = append(ids, metaId)
 
+	eSchemaId, err := f.stateStore.JobStore.EnqueueJob(ctx, job.Job{
+		Dir: dir,
+		Func: func(ctx context.Context) error {
+			return jobs.PreloadEmbeddedSchema(ctx, f.logger, schemas.FS,
+				f.store, f.stateStore.ProviderSchemas, path)
+		},
+		DependsOn:   job.IDs{metaId},
+		Type:        operation.OpTypePreloadEmbeddedSchema.String(),
```

Comment:
I think we should introduce a new job type, so that these jobs won't get merged with module preload schema jobs.